### ISSUE TITLE
Support Carbon v11

### ISSUE THREAD

**Issue by photodow** [carbon-design-system/devtools#158]
Carbon v11 is coming. With that I understand are some updates to tokens and some things under the hood.

To do:
- [ ] We need to one evaluate what is needed to adopt v11 in Carbon devtools.
- [ ] Maybe find a way to provide both v10 and v11 as options while teams are in transition. #6 
- [ ] then execute on it.

https://www.carbondesignsystem.com/whats-happening/v11-release/

**Comment by MichCest** [comment]
@photodow @tay1orjones is there any plan for timing to update this plugin for carbon v11?

**Comment by motou** [comment]
Yes, I am planning to provide v11 support in the coming months. Any help or suggestions would be appreciated.

**Comment by smith558** [comment]
> Yes, I am planning to provide v11 support in the coming months. Any help or suggestions would be appreciated.

Is this still on the roadmap? 🙂

**Comment by Laloses** [comment]
Recently Devtools has stopped working on my projects on v11 @carbon/react > 1.56.0
